sentry/templates/configmap-sentry.yaml

```diff
@@ -468,51 +406,76 @@ data:
     SENTRY_OPTIONS['mail.host'] = os.getenv("SENTRY_EMAIL_HOST", {{ .Values.mail.host | quote }})
     SENTRY_OPTIONS['mail.from'] = os.getenv("SENTRY_EMAIL_FROM", {{ .Values.mail.from | quote }})
 
+    #######################
+    # MaxMind Integration #
+    #######################
+
+    {{- if .Values.geodata.path }}
+    GEOIP_PATH_MMDB = {{ .Values.geodata.path | quote }}
+    {{- end }}
+    
     #########################
     # Bitbucket Integration #
     #########################
-
+    
     # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
     # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
 
-    #########
-    # Relay #
-    #########
-    SENTRY_RELAY_WHITELIST_PK = []
-    SENTRY_RELAY_OPEN_REGISTRATION = True
-
-    #######################
-    # OpenAi Suggestions #
-    #######################
+    ##############################################
+    # Suggested Fix Feature / OpenAI Integration #
+    ##############################################
+    
+    # See https://docs.sentry.io/product/issues/issue-details/ai-suggested-solution/
+    # for more information about the feature. Make sure the OpenAI's privacy policy is
+    # aligned with your company.
     
+    # Set the OPENAI_API_KEY on the .env or .env.custom file with a valid
+    # OpenAI API key to turn on the feature.
     OPENAI_API_KEY = os.getenv("OPENAI_API_KEY", "")
+    
     if OPENAI_API_KEY:
       SENTRY_FEATURES["organizations:open-ai-suggestion"] = True
+    
+    ##############################################
+    # Content Security Policy settings
+    ##############################################
+    
+    if "csp.middleware.CSPMiddleware" not in MIDDLEWARE:
+        MIDDLEWARE = ("csp.middleware.CSPMiddleware",) + MIDDLEWARE
+    # CSP_REPORT_URI = "https://{your-sentry-installation}/api/{csp-project}/security/?sentry_key={sentry-key}"
+    CSP_REPORT_ONLY = True
+    
+    # optional extra permissions
+    # https://django-csp.readthedocs.io/en/latest/configuration.html
+    # CSP_SCRIPT_SRC += ["example.com"]
 
-{{- if .Values.metrics.enabled }}
+    {{- if .Values.metrics.enabled }}
     SENTRY_METRICS_BACKEND = 'sentry.metrics.statsd.StatsdMetricsBackend'
     SENTRY_METRICS_OPTIONS = {
         'host': '{{ template "sentry.fullname" . }}-metrics',
         'port': 9125,
     }
-{{- end }}
+    {{- end }}
 
-{{- if .Values.slack.existingSecret }}
+    {{- if .Values.slack.existingSecret }}
     #########
-    # SLACK #
+    # Slack #
```

Comment:
I left these in, but I think they may actually be duplicative because of the configuration in `config.yml` [(which is where it lives in self hosted)](https://github.com/getsentry/self-hosted/blob/master/sentry/config.example.yml#L109)

i can remove these if it's covered by having it in the other place